website/src/app/kb/reference/glossary/readme.mdx

```diff
@@ -0,0 +1,33 @@
+# Glossary of terms
+
```

Comment:
Alphabetize, also:

```suggestion

**Account slug**: Your organization's tenant name, unique across all Firezone customers, e.g. `international-widget-corp-inc`

**Firezone ID**: AKA "Device ID", a non-empty string that identifies Clients and Gateways, unique within your organization, e.g. `johns-laptop` or `01HQP2K1AM41XWWNBVZS39N062`

**Admin portal**: AKA "portal" or "Web app", the central control plane where admin users create, manage, and delete Resources, Gateways, and Service Tokens for headless clients.
```

Hope the noob perspective helps lol

Reply:
I'll save `FIREZONE_ID` for an upcoming env var reference I'm planning to write.